testing/bdd/templates/complete_document.xml

```diff
@@ -2,8 +2,7 @@
 <tt:tt
         ebuttp:sequenceIdentifier="TestSequence"
         ebuttp:sequenceNumber="1"
-        ttp:clockMode="local"
-        ttp:timeBase="clock"
+        ttp:timeBase="media"
```

Comment:
Please could you keep this file as is, since other tests depend on it, and create a new file with media timebase? I'd be happy to rename this `complete_document_clock.xml` and have the new document be `complete_document_media.xml` and change `passive_nodes_shall_not_modify_document.feature` to include both files in the examples, since they should both pass.